XenModel/XenModel.csproj

```diff
@@ -353,6 +354,9 @@
     <Compile Include="XenAPI\pgpu_dom0_access.cs" />
     <Compile Include="XenAPI\pool_allowed_operations.cs" />
     <Compile Include="XenAPI\primary_address_type.cs" />
+    <Compile Include="XenAPI\PVS_farm.cs" />
+    <Compile Include="XenAPI\PVS_proxy.cs" />
+    <Compile Include="XenAPI\PVS_server.cs" />

```

Comment:
I'm curious; these files were added in a previous commit/ticket, right? (The bindings update one) So why have these lines just appeared here?